meetings/2024-09-27.md

```diff
@@ -134,8 +139,8 @@ URL: https://lists.w3.org/Archives/Public/public-lws-wg/2024Sep/0002.html
+* SC: I find this portability topic comes up a lot but an important distinction is what web architecture says about moving resources around, it's very clear about redirects. Another topic about moving data around. AFAIK those are separate things: moving URIs around than having a copy of your data in a new storage. The URI aspect is covered under webarch, copying/archiving your storage could be exporting a dump of your storage and having mechanism on the new storage to take that and extract, for example. There is even `HTTP COPY` (from WebDAV) and we had that in our radar ( https://github.com/solid/specification/issues/19 ). Copying the data is different from saying this resource identified by this URI is moved to another location. The harder parts of the problem is coming up with URI persistence policies and enforcing. What kind of assurances will there be that they can continue to use that when moved to another storage? Some of that is the policy work and not so much the architectural decision on how protocols and interfaces speak to each other. If we can find persistence policies that'd be the answer to the problem. The copying the data seems more complex technically but it's a lot simpler than handling the persistence issues. 
 * eP: what I'd like to get out is a strategy for UCs. For me the UCs are real-world, as opposed to the solutions. Working with the Food Network, open source software. It'd be good to connect with existing communities with user bases. Easy to experiment with. Wha thtye do but also using Solid. We should be hte last peopel to come pu with UCs. Get UCs from facing the problems. We used the solid forum, which is open source also. social web also wants to to do threads. We can showcase how this UC can overlap with ??? If we get a UC, we extract requirements, and some may be out of scope. Since some of the rquirements may be outside of hte scope, ??? then some will be for the WG and others left for the CG. Real challenge is to get the UCs from out there.
 * RG: To answer HZ's UC quickly using SC's answer. Goign got flip SC's answer. I think this is not a problem for just Solid or LWS. Maybe for Web platform, the reality is 1B people in a country like India will not be abl to own a domain name. Portability is not always going ot work, and I don't tihnk we are in a place to answer it.
```

Comment:
```suggestion
* eP: What I'd like to get out is a strategy for UCs. For me, the UCs are real-world, as opposed to the solutions. Working with the Food Network, open source software. It'd be good to connect with existing communities with user bases. Easy to experiment with. What they do, but also using Solid. We should be the last people to come up with UCs. Get UCs from facing the problems. We used the Solid forum, which is also open source. Social web also wants to to do threads. We can showcase how this UC can overlap with ??? If we get a UC, we can extract requirements, and some may be out of scope. Since some of the requirements may be outside of the WG scope, ??? then some will be for the WG and others left for the CG. Real challenge is to get the UCs from out there.
* RG: To answer HZ's UC quickly, using SC's answer. Going got flip SC's answer. I think this is not a problem for just Solid or LWS. Maybe for Web platform, the reality is 1Billion people in a country like India will not be able to own a domain name. Portability is not always going to work, and I don't think we are in a place to answer it.
```